autocompleter/base.py

```diff
@@ -1145,7 +1145,9 @@ def update_all(self, clear_cache=True):
         """
         Update all modified objects within all the ac's providers
         """
-        for provider_class in self._get_all_providers_by_autocompleter():
+        if not (provider_classes := self._get_all_providers_by_autocompleter()):
+            return
```

Comment:
Can we also log an error so that the person knows they messed up calling this, otherwise this would silently not update without anyone knowing.